docs/product/tracing/index.mdx

```diff
@@ -0,0 +1,44 @@
+
+### What's a Trace?
+
+Each trace has a string of globally unique characters called a trace ID, that's shared by both frontend and backend services they help connect all the actions that take place to make an event such as a button click or a page scroll. Each trace is composed of multiple spans.
+
+### What's a Span?
+A span is a named, timed operation that represents a part of the application workflow. It provides granular insights into specific tasks, like API requests or database queries. Multiple spans are pieced together to create a trace. providing a comprehensive view of its path, timing, and interactions for troubleshooting and performance analysis.
+
+### What's a Transaction?
+
+Transactions are a Sentry-specific unit of measurement that references any event that users send to Sentry. In addition to standard event fields, transactions also contains child spans and have a unique transaction ID.
+
+## How to Use Tracing in Sentry
+
+To get the most out of tracing in Sentry, you'll need to instrument both the frontend and backend of your application with Sentry's SDK. Once this is done, there are several places where you can view and interact with tracing data:
+
+### Traces to Trace View
+
+You can view all the traces in your organization by going to the [Traces](https://sentry.io/orgredirect/organizations/:orgslug/traces/) page in Sentry. You'll see a chart and a list of traces and be able to see at-a-glance how long each trace took to complete and the number of spans it contains.
+
+If you want more information, click on any trace ID. This will take you to the [Trace View](https://sentry.io/orgredirect/organizations/:orgslug/traces/traceid/) page, which provides a more detailed view of the trace and its spans. Here, you'll see a waterfall view of the spans in the trace, which will show you how long each service and process took to complete and may give you an indicator of where a problem may be coming from. Learn more about the Sentry [Trace View](/concepts/key-terms/tracing/trace-view/).
+
+Alternatively, you can use the search bar to look for traces by name, project, or other criteria. You can also use the tags section to filter traces by specific tags. Learn more about the Sentry [Trace Explorer](/product/explore/traces/).
+
+### Performance to Trace View
+
+If you're interested in a high-level view of your application's performance, you can start from the [Performance](https://sentry.io/orgredirect/organizations/:orgslug/performance/) page in Sentry where you'll see a list of transactions. Clicking on the transaction ID will take you to the Trace View page for that transaction. Learn more about using Sentry's [Performance Monitoring](/product/performance/).
```

Comment:
Should we mention Insights here? That's also high level and takes you to the trace view, and that's a newer workflow that we would want to highlight over the Performance landing page. The Performance landing page is still good for seeing a list of pages/endpoints however.

Reply:
The "traces to trace view" section is insights :) unless I'm misunderstanding and you're thinking of a different workflow?